masterbase/guards.py

```diff
@@ -85,3 +85,12 @@ async def valid_session_guard(connection: ASGIConnection, _: BaseRouteHandler) -
         Server.query_from_params(api_key, converted_fake_ip, fake_port)
     except KeyError:
         raise NotAuthorizedException("Cannot accept data from a non-existent gameserver!")
+
+
+async def valid_target_guard(connection: ASGIConnection, _: BaseRouteHandler):
+    """Verify the existence of a target_steam_id, if applicable (otherwise, no-op)."""
+    params = connection.query_params
+    if "target_steam_id" in params:
+        return account_exists(params["target_steam_id"])
+    else:
+        return True
```

Comment:
shouldnt need to test for existence of this param, litestar will throw a bad request if a user sends a post without that param in the query params as it is marked as a `str`.

Reply:
rev https://github.com/MegaAntiCheat/masterbase/pull/58/commits/c41c087075f1ef2b9ab3e14da6c526dc7eb04e58